src/TestCaseFilters/GitDirtyTestCaseFilter.php

```diff
@@ -32,6 +32,7 @@ public function accept(string $testCaseFilename): bool
         assert(is_array($this->changedFiles));
 
         $relativePath = str_replace($this->projectRoot, '', $testCaseFilename);
+        $relativePath = str_replace(DIRECTORY_SEPARATOR, '/', $relativePath);
```

Comment:
@dansysanalyst Can you test this change please?

Reply:
It works as expected.

![CleanShot 2023-03-22 at 01 49 56@2x](https://user-images.githubusercontent.com/79267265/226773874-f2feb511-e3a3-438b-940a-2969c2c6f37a.png)